general/src/processor/attrs.rs

```diff
@@ -69,12 +69,12 @@ impl FromStr for SelectorSpec {
             "freeform" | "email" | "sensitive" | "text" => {
                 return Ok(SelectorSpec {
                     path: vec![SelectorPathItem::Type(ValueType::String)],
-                })
+                });
```

Comment:
This makes me question if I have the wrong rustfmt installed. Let's see what CI says